src/languages/id-ID.js

```diff
@@ -402,14 +402,14 @@ export default {
     enableProxyDescription: "Proxy berikut tidak akan berdampak ke monitor hingga diaktifkan. Anda dapat mengontrol menonaktifkan sementara proxy dari semua monitor dengan status aktivasi.",
     setAsDefaultProxyDescription: "Proxy berikut akan diaktifkan sebagai bawaan untuk monitor baru. Anda masih dapat menonaktifkan proxy secara terpisah untuk setiap monitor.",
     "Certificate Chain": "Certificate Chain",
-    Valid: "Sahih",
+    Valid: "Valid",
```

Comment:
I don't speak Indonesian, is the English `Valid` more commonly used / more correct than the previous `Sahih`

Reply:
yes valid is commonly used, sahih more arabic